### ISSUE TITLE
Normalization/ Rescaling

### ISSUE THREAD

**Issue by kamehta2** [EHWUSF/HS68_2018_Project_1#5]
The normalization is the process of rescaling attributes to the range of 0 to 1. This means that the largest value for each attribute is 1 and the smallest value is 0. I propose that it is a good technique to use when there are different units in the data and also when some attributes which have a wide range of values.

**Comment by nirveshk** [comment]
Hi kinjal,
I believe this is too general. I think there is already syntax/function that accomplishes your proposed idea. May be proposing a precised idea of different approach to normalization would be a way to go :-)
Please feel free to correct me if I am wrong.

**Comment by kamehta2** [comment]
@nirveshk  Yes, it is little general but we can think of making some module which can suggest if the dataset needs normalization/ rescaling or not. And if yes then it can do it for you. 
For example, It can check if there is a wide range of values in the dataset and we can set some kind of limit or something and then it will give you rescaled dataset.

**Comment by kamehta2** [comment]
And I will check some more things about this.
